src/getData.js

```diff
@@ -0,0 +1,25 @@
+const ulWrap = document.querySelector('.score-list');
+
+let arrOfScores = [];
+
+const previousData = JSON.parse(localStorage.getItem('storedScores'));
+const previousInput = JSON.parse(localStorage.getItem('storedInput'));
```

Comment:
- [ ] Let's use the [Leaderboard API](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) and the `fetch` method coupled with `async` and `await` to send and receive data. Usage of localStorage here is optional.